src/DependencyInjection/DoctrineEncryptExtension.php

```diff
@@ -58,9 +59,21 @@ public function load(array $configs, ContainerBuilder $container): void
             $loader->load('services_with_secret.yml');
         }
 
-        // Remove usage of AttributeAnnotationReader when using php < 8.0
-        if (PHP_VERSION_ID < 80000) {
-            $container->setAlias('ambta_doctrine_annotation_reader','annotations.reader');
+        // Symfony 1-3
+        if (Kernel::MAJOR_VERSION < 5) {
+            throw new \RuntimeException('doctrineencryptbundle/doctrine-encrypt-bundle expects symfony-version >= 4.1!');
```

Comment:
This message should be updated.

Reply:
Is this necessary? The `composer.json` will prevent installing with old versions of Symfony.

Reply:
Not really, more of a sanity check.